python/lsst/meas/astrom/anetAstrometry.py

```diff
@@ -48,6 +48,11 @@ class ANetAstrometryConfig(pexConfig.Config):
     rejectIter = pexConfig.RangeField(dtype=int, default=3, doc="Rejection iterations for Wcs fitting",
                                       min=0)
 
+    @property
+    def refObjLoader(self):
+        """An alias, for a uniform interface with the standard AstrometryTask"""
+        return self.solver
+
```

Comment:
The change triggering the problem is fairly recent (about a month ago; DM-5686), and I suspect no-one has used our astrometry.net solver since that went in.